AUTHORS

```diff
@@ -505,3 +505,4 @@ a license to everyone to use it as detailed in LICENSE.)
 * Paul Peny <pmpp.pub@gmail.com>
 * Prashanth Nethi <prashant@adobe.com>
 * Max Weisel <max@maxweisel.com>
+* Georg Rottensteiner <georg@georg-rottensteiner.de>
```

Comment:
this may need to be rebased, as this and other changes look irrelevant?

Reply:
Done.